src/include/miopen/solver.hpp

```diff
@@ -807,6 +807,8 @@ struct PerformanceConvMlirIgemm : Serializable<PerformanceConvMlirIgemm>
     int GemmNPerThread;
     bool use_spare_set;
 
+    static const PerformanceConvMlirIgemm& GetHeuristicInitRequest();
```

Comment:
[Recommendation] This is implementation detail of MLIR solvers, so I would prefer to make it private. This is not possible with this simple design, so let's use the "Mlir" prefix to emphasize that this method is MLIR-specific.
```suggestion
    static const PerformanceConvMlirIgemm& GetMlirHeuristicInitRequest();
```
In addition, "Get" can be removed (it's up to you).

Reply:
I made it to be `MlirHeuristicInitRequest`